sbol/__init__.py

```diff
@@ -25,8 +25,7 @@
 from .moduledefinition import ModuleDefinition
 from .participation import Participation
 from .partshop import PartShop
-from .property import LiteralProperty
-from .property import TextProperty
+from .property import LiteralProperty, TextProperty, IntProperty
```

Comment:
Items out of (alphabetical) order are like nails on a chalkboard.

```suggestion
from .property import IntProperty
from .property import LiteralProperty
from .property import TextProperty
```